fe/fe-core/src/main/java/com/starrocks/catalog/IcebergResource.java

```diff
@@ -48,6 +48,9 @@ public class IcebergResource extends Resource {
     @SerializedName(value = "catalogImpl")
     private String catalogImpl;
 
+    @SerializedName(value = "fsS3AuthInfo")
```

Comment:
After 2.5 version, we won't update external tables and resources, but use a external catalog instead. In terms of compatibility, we will create an internal catalog for dla resources to use. Therefore, plz don't persist any information into the resource. If necessary, it can be put into properties.

Reply:
move it to resource's properties.

Reply:
move it to resource's properties.



> After 2.5 version, we won't update external tables and resources, but use a external catalog instead. In terms of compatibility, we will create an internal catalog for dla resources to use. Therefore, plz don't persist any information into the resource or update dla resource. If necessary, it can be put into properties.